controllers/v1/tasks.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * name : tasks.js
+ * author : Aman
+ * created-date : 18-Nov-2020
+ * Description : Tasks related information.
+ */
+
+// Dependencies
+const tasksHelper = require(MODULES_BASE_PATH + "/tasks/helper");
+
+/**
+   * Tasks
+   * @class
+*/
+
+module.exports = class Tasks {
+
+    static get name() {
+        return "tasks";
+    }
+
+    /**
+    * @api {post} /improvement-project/api/v1/tasks/status/:projectId
```

Comment:
@aman-tunerlabs - All these APIs need to be inside the project. It cannot be outside independent

Reply:
@aks30  we discuss the api endpoint and build in tasks. Here's the documentation - https://docs.google.com/document/d/16QW6teqHVohYT6kz29y0F2uOyi70LI_14j-j5kxiT6g/edit